.idea/compiler.xml

```diff
@@ -18,6 +18,5 @@
         <processorPath useClasspath="true" />
       </profile>
     </annotationProcessing>
-    <bytecodeTargetLevel target="1.6" />
   </component>
```

Comment:
not really sure what's going on here. language level is specified in the gradle file, I'm suprised it's removed here.